services/QuillLMS/app/assets/stylesheets/pages/about-us-pages.scss

```diff
@@ -94,6 +94,11 @@
     width: 604px;
   }
 
+  #mit-logo {
+    height: 58%;
```

Comment:
I would be inclined to use pixels instead of a percentage here, to avoid unexpected changes if the container height changes? But NBD if there's a reason to prefer percentage.